### PR TITLE
Portfolio popup Window Project

### PR DESCRIPTION
**Pull request by Carshy**
In this project milestone, I set up a popup window for my portfolio project. During the project, I implemented the solution using JavaScript, HTML DOM elements and CSS. I learned the following concept:
* How to implement a popup window using JavaScript
* Learned HTML DOM elements
* How to use JavaScript functions to access DOM elements
* How to use JS objects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,96 +56,65 @@ <h2 class="connect"><a href="#">LET’S CONNECT</a></h2>
...
-                                <li class="JavaScript-language">javaScript</li>
-                            </ul>
-                            <div class="project-btn">
-                                <button class="details-button" type="button">See Project</button>
-                            </div>  
+        
+            <div id="details-container" class="main-container">
+
+                <!-- start of Pop up Window Container -->
+                <div class="popup-details">
```
Comment (tufoinnkuo10) [inline index.html:63]: - [ ] kindly make the mobile popup full width and height.  also, make the see source and see live button available
your version 👇🏾 
Screenshot (188)_LI
the figma version 👇🏾 

Screenshot (186)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Carshy 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ array works well
✅ responsive
✅ popup works well

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Carshy** [comment]
Made changes as requested by the previous reviewer. @tufoinnkuo10 thanks for reviewing my code

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Carshy,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] kindly style the logo and the hamburger in the header.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**